### ISSUE TITLE
Image resolution read is upside down

### ISSUE THREAD

**Issue by daniherculano84** [SixLabors/ImageSharp#462]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of ImageSharp
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
I'm uploading to my Net Core 2.0 web app a vertical image with the resolution 3456x5184 (width x height) and I see that the Width and Height Image properties are upside down, so Width is 5184 and Height is 3456. That resolution confuses my application and treats that image as an image in landscape orientation.

### Steps to Reproduce
This is my code:
```
using (Image img = new Image(pathDst))
 {
       myimg.HeightImg = img.Height.ToString(); //WRONG VALUE
       myimg.WidthImg = img.Width.ToString(); //WRONG VALUE
       myimg.FormatImg = img.CurrentImageFormat.MimeType;
}
```

### System Configuration

- ImageSharp version: 0.9.0
- Environment (Operating system, version and so on): W10 x64
- .NET Framework version: ASP NET Core 2.0 Web App.

**Comment by JimBobSquarePants** [comment]
Hi there, 

That version number looks suspicious, have you downloaded the official SixLabors package? It should be beta2. 

Cheers

James

**Comment by daniherculano84** [comment]
Hi Jim,
I've changed RevStackCore-ImageSharp 0.9.0 DLL to SixLabors.ImageSharp 1.0.0-beta0002.
The problem persists.

The new code is
```
using (var img = Image.Load(pathDst))
{
       myimg.HeightImg = img.Height.ToString(); //WRONG VALUE
       myimg.WidthImg = img.Width.ToString(); //WRONG VALUE
       //myimg.FormatImg = ??
}
```

**Comment by JimBobSquarePants** [comment]
I'll need to see the image in question before I can make any judgement so could you please upload it?

If you're looking for the imageformat btw there's an overload for Load that has an out parameter.

**Comment by daniherculano84** [comment]
This is the image:
https://ibb.co/j0KgTx

Thank you

**Comment by JimBobSquarePants** [comment]
Hi @daniherculano84 

I posted an answer hours ago but somehow GitHub didn't actually save it. 🙁 

I tested your image with the latest nightly and although I don't recall any changes between that and the beta (I haven't tested against the beta) you will see from the screenshot that the image dimensions are reported correctly. 

![size](https://user-images.githubusercontent.com/385879/36045981-8e1f8932-0e2b-11e8-8a64-5576bfe9c57d.jpg)

I would download the nightly from our MyGet repo listed in the readme and give that a go.

Cheers

James
